WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md

```diff
@@ -35,9 +35,9 @@ If you enable the **Try Next Closest Site** setting, DC Locator uses the followi
 - If no domain controller is available in the same site, try to find a domain controller in the next closest site. A site is closer if it has a lower site-link cost than another site with a higher site-link cost.  
 - If no domain controller is available in the next closest site, try to find any domain controller in the domain.  
 
-By default, DC Locator does not consider any site that contains a read-only domain controller (RODC) when it determines the next closest site. In addition, because only  Windows Server 2008  and  Windows Server 2008 R2  domain controllers support the next closest site functionality, when the client gets a response from a domain controller that runs an earlier version of Windows Server, the DC Locator behavior is the same as when then setting is not enabled.  
+By default, DC Locator does not consider any site that contains a read-only domain controller (RODC) when it determines the next closest site. In addition, when the client gets a response from a domain controller that runs an version earlier than Windows Server 2008, the DC Locator behavior is the same as when then setting is not enabled.  
```

Comment:
that runs an version earlier than Windows Server 2008, > that runs a version earlier than Windows Server 2008,

Reply:
done